AlfiyaSiddique/TastyTrails
# Loader while signup and login

## What does this PR do?
This PR adds loader when user clicks login and signup till backend processes and when result comes either error or success it is disabled

I have Loader component and i apply it to screen as loading starts and set it false as result comes
Fixes # (issue)
fixes #357 


## How should this be tested?



![image](https://github.com/user-attachments/assets/8100f00f-3d7e-41b2-a53c-21c069f08fe4)
No responsive issue
![image](https://github.com/user-attachments/assets/a8f09351-2872-488e-8a65-9b26ad14b610)
Loader looks something like this

- Go to login Page and fill details and click login
- Go to signup page and fill details and click signup and you will see the change till processing occur
For better experience you can ensure your network is slow so that you can properly see the effect as backend will take more time to process.
## Checklist



### Required

- [ x] Filled out the "How to test" section in this PR
- [ x] Read Contributing guide
- [ x] Self-reviewed my own code
- [x ] Checked for warnings, there are none
- [ x] Removed all `console.logs`
- [x ] The changes don't cause any responsiveness issues
